include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -264,6 +338,18 @@ MGConstrainedDoFs::have_boundary_indices () const
 }
 
 
+
+
+inline
+const ConstraintMatrix &
+MGConstrainedDoFs::get_level_constraint_matrix (unsigned int level) const
```

Comment:
make the argument `const` to indicate that it's an input argument. if necessary, do the same in the function declaration in the .h file.